conn.go

```diff
@@ -36,12 +37,31 @@ func (d *Driver) Open(dsn string) (driver.Conn, error) {
 	return &Conn{h: h}, nil
 }
 
-func (c *Conn) Close() error {
+func (c *Conn) Close() (err error) {
+	if c.tx != nil {
```

Comment:
I found that if the transaction is not closed `api.SQLFreeHandle` returns the error "SQLFreeHandle: {HY010} [Microsoft][ODBC Driver Manager] Function sequence error".